### ISSUE TITLE
[Linux] Missing packages in environment.yaml

### ISSUE THREAD

**Issue by GuidoBartoli** [lllyasviel/Fooocus#878]
**Describe the problem**
During installation from scratch in Conda on an Ubuntu-based system, two required packages (`numpy` and `Pillow`) are not installed automatically.

**Full Console Log**
```
$ git clone https://github.com/lllyasviel/Fooocus.git
$ cd Fooocus
$ conda env create -f environment.yaml
$ conda activate fooocus
$ pip install pygit2==1.12.2
$ python entry_with_update.py
Already up-to-date
Update succeeded.
Traceback (most recent call last):
  File "/home/bartoli/Projects/Fooocus/entry_with_update.py", line 46, in <module>
    from launch import *
  File "/home/bartoli/Projects/Fooocus/launch.py", line 10, in <module>
    from modules.path import modelfile_path, lorafile_path, vae_approx_path, fooocus_expansion_path, \
  File "/home/bartoli/Projects/Fooocus/modules/path.py", line 5, in <module>
    import modules.sdxl_styles
  File "/home/bartoli/Projects/Fooocus/modules/sdxl_styles.py", line 5, in <module>
    from modules.util import get_files_from_folder
  File "/home/bartoli/Projects/Fooocus/modules/util.py", line 1, in <module>
    import numpy as np
ModuleNotFoundError: No module named 'numpy'
$ conda install numpy
$ python entry_with_update.py
Already up-to-date
Update succeeded.
Traceback (most recent call last):
  File "/home/bartoli/Projects/Fooocus/entry_with_update.py", line 46, in <module>
    from launch import *
  File "/home/bartoli/Projects/Fooocus/launch.py", line 10, in <module>
    from modules.path import modelfile_path, lorafile_path, vae_approx_path, fooocus_expansion_path, \
  File "/home/bartoli/Projects/Fooocus/modules/path.py", line 5, in <module>
    import modules.sdxl_styles
  File "/home/bartoli/Projects/Fooocus/modules/sdxl_styles.py", line 5, in <module>
    from modules.util import get_files_from_folder
  File "/home/bartoli/Projects/Fooocus/modules/util.py", line 7, in <module>
    from PIL import Image
ModuleNotFoundError: No module named 'PIL'
$ conda install Pillow
$ python entry_with_update.py
Already up-to-date
Update succeeded.
Python 3.10.13 (main, Sep 11 2023, 13:44:35) [GCC 11.2.0]
Fooocus version: 2.1.779
```

**System info**
```
             ...-:::::-...                 bartoli@HPZ8 
          .-MMMMMMMMMMMMMMM-.              ------------ 
      .-MMMM`..-:::::::-..`MMMM-.          OS: Linux Mint 21.2 x86_64 
    .:MMMM.:MMMMMMMMMMMMMMM:.MMMM:.        Host: HP Z8 G4 Workstation 
   -MMM-M---MMMMMMMMMMMMMMMMMMM.MMM-       Kernel: 5.15.0-88-generic 
 `:MMM:MM`  :MMMM:....::-...-MMMM:MMM:`    Uptime: 3 days, 1 hour, 31 mins 
 :MMM:MMM`  :MM:`  ``    ``  `:MMM:MMM:    Packages: 2558 (dpkg), 8 (flatpak) 
.MMM.MMMM`  :MM.  -MM.  .MM-  `MMMM.MMM.   Shell: bash 5.1.16 
:MMM:MMMM`  :MM.  -MM-  .MM:  `MMMM-MMM:   Resolution: 1680x1050, 1920x1200 
:MMM:MMMM`  :MM.  -MM-  .MM:  `MMMM:MMM:   DE: Cinnamon 5.8.4 
:MMM:MMMM`  :MM.  -MM-  .MM:  `MMMM-MMM:   WM: Mutter (Muffin) 
.MMM.MMMM`  :MM:--:MM:--:MM:  `MMMM.MMM.   WM Theme: Mint-Y-Dark-Aqua (Mint-Y) 
 :MMM:MMM-  `-MMMMMMMMMMMM-`  -MMM-MMM:    Theme: Mint-Y-Dark-Aqua [GTK2/3] 
  :MMM:MMM:`                `:MMM:MMM:     Icons: Mint-Y-Sand [GTK2/3] 
   .MMM.MMMM:--------------:MMMM.MMM.      Terminal: gnome-terminal 
     '-MMMM.-MMMMMMMMMMMMMMM-.MMMM-'       CPU: Intel Xeon Silver 4214Y (24) @ 3.200GHz 
       '.-MMMM``--:::::--``MMMM-.'         GPU: NVIDIA Quadro RTX 6000/8000 
            '-MMMMMMMMMMMMM-'              Memory: 10108MiB / 31795MiB 
               ``-:::::-``
```

**Current workaround**
The manual solution is to install both packages in Conda environment: `conda install numpy Pillow`

**Problem solution**
Add missing packages in `environment.yaml`:
```
name: fooocus
channels:
  - defaults
dependencies:
  - python=3.10
  - pip=23.0
  - packaging
  - numpy
  - Pillow
```

**Comment by guoreex** [comment]
It seems that the problem still exists.

I installed the version 1 month ago and haven't used it. Today, I started with "python3 entry_with_update.py" and prompted me whether to follow anything. After agreeing, there was a problem with GuidoBartoli.

`$ pip3 uninstall pillow
$ pip3 uninstall PIL
$ pip3 install image`
After executing the above command on ubuntu, I started fooocus again, and it worked normally.
Sorry, I'm not a professional programmer. I didn't know if I should deal with it like this.
